code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -52,6 +52,8 @@
 			audible_message("<span class='boldwarning'>An unearthly screaming sound resonates throughout Reebe!</span>")
 			for(var/V in GLOB.player_list)
 				var/mob/M = V
+				if(!M)
+					continue
```

Comment:
We don't do this check for GLOB.player list, it's treating the symptom of the cause. Nothing in that list should be null